blog/2022-11-09-apache-monitoring.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: apache-monitoring
+title: Apache Monitoring
+authors: shyam
+tags: [how-to, monitor, troubleshoot, Apache, web server]
```

Comment:
```suggestion
tags: [how-to, monitor, troubleshoot, Apache, web server]
keywords: [how-to, monitor, troubleshoot, Apache, web server]
```

Reply:
i think we need to add these keywords for seo keywords in addition to just the tags. not 100% sure but no harm for now